FamilyNet/FamilyNet/Models/Orphan.cs

```diff
@@ -20,6 +20,7 @@ public override void CopyState(Person sender)
         {
             base.CopyState(sender);
             Orphan orphanSended = sender as Orphan;
+            Avatar = orphanSended.Avatar;
```

Comment:
it`s wrong, look at controller logic, u rewritten to null avatar if user don`t change avatar